### DISCUSSION TITLE
Problem on c0-r0

### DISCUSSION THREAD

**Discussion by nicolasvair** [shanteacontrols/OpenDeck#233] [General]
Hi,

I'm currently mounting a board, and it seems I can't use the c0-r0 connection, when I connect them, the response time is very high (around 1s). I have the same behavior with c1-r0. Other connections are fine (c0-r1 to r7, c1-r1 to r7). 

I'm using the latest version of open deck board. Any idea ? Am I doing something wrong ? 

Thank you for your help !

**Comment by paradajz** [comment]
Please specify the exact version of the board you have.

**Comment by nicolasvair** [comment]
BoardOpenDeck M v2.2Firmwarev7.4.6

**Comment by nicolasvair** [comment]
I just updated the board to firmware 7.5.4 just in case, it seems the problem is still present

**Comment by paradajz** [comment]
I've tried it on my board and I cannot reproduce (latest firmware as of this writing: v7.5.4).

**Comment by nicolasvair** [comment]
Wow this morning the problem disapeared, could it be somehting related to electrostatic issue ? I remember having this on arduino mega on another project using open deck. Any idea / best practice to reduce this kind of issues ?
